schemas/v1.0/azure.yaml.json

```
@@ -148,6 +148,22 @@
                 },
                 "required": ["project"]
             }
+        },
+        "pipeline": {
+            "type": "object",
+            "title": "Definition of continuous integration pipeline",
+            "additionalProperties": true,
+            "properties": {
+                "provider":{
+                    "type": "string",
+                    "title": "Type of pipeline provider",
+                    "description": "Optional. The pipeline provider to be used for continuous integration. (Default: GitHub)",
```

```suggestion
                    "description": "Optional. The pipeline provider to be used for continuous integration. (Default: github)",
```

For case correctness

---

what do you mean with case correctness?  the input is lowercased, so we don't need customers to worry about case correctness